main/templates/main/tenants.html

```diff
@@ -8,9 +8,10 @@
 		box-sizing: content-box;
 	}
 </style>
+<script src="{% static "main/sorttable.js" %}"></script>
```

Comment:
I think this line should go in the "script" block (at the bottom of the file).
Apart from that we're good to go :+1: